### PR TITLE
Added option to change prefix in config

### PR DESCRIPTION
**Pull request by floppydisk05**

### CODE REVIEW

**Comment by Starman0620** [comment]
The prefix needs to be changed in the DMSystem aswell as HauntSystem. Lines 62 and 49 respectively

**Comment by floppydisk05** [comment]
dang it

**Comment by floppydisk05** [comment]
sec

**Comment by floppydisk05** [comment]
there

File: Source/Misc/HauntSystem.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public static void Init()
                 }
                 // Host to target
                 else if(chats.FirstOrDefault(x => x.host.Id == args.Channel.Id) != null) {
-                    if(args.Message.Content.StartsWith("."))
+                    if(args.Message.Content.StartsWith($"{Bot.config.prefix"))
```
Comment (Starman0620) [inline Source/Misc/HauntSystem.cs:49]: "{Bot.config.prefix" 

You need a closing }... or just replace the string entirely with just "Bot.config.prefix"

Comment (floppydisk05) [reply Source/Misc/HauntSystem.cs:49]: I'm a fucking donut

**Pull request merged**